eng/pipelines/ci.yml

```diff
@@ -56,6 +56,10 @@ jobs:
           OSName: "Linux"
           OSVmImage: "MMSUbuntu20.04"
           PythonVersion: "3.10"
+        Linux_Python311:
+          OSName: "Linux"
+          OSVmImage: "MMSUbuntu22.04"
+          PythonVersion: "3.11"
```

Comment:
I'm wondering we can keep `3.10` for maybe half a year? Because `3.11` has just been released and having both of them tested can help use quickly know potentially gap between these two. 
While the cost for us is that the total pipeline execution time is increased for 1~2 minutes.

Reply:
I think we can keep both for now, but yeah we should eventualy remove 3.10.